tests/functional/spec/cookie_banner.spec.js

```diff
@@ -0,0 +1,32 @@
+import InitialPage from "../generated_pages/checkbox/mandatory-checkbox.page";
+
+describe('Given I open runner"', () => {
+  it("When I visit the root address then the cookie banner shouldn‘t be displayed", () => {
```

Comment:
This implies this is only true if you visit the root, but it more about `Given I am not authenticated and have no cookie, When I visit a page in runner, Then the cookie banner shouldn‘t be displayed`